### DISCUSSION TITLE
How the image is created in the Gopro?

### DISCUSSION THREAD

**Discussion by Pax709** [gopro/labs#898] [Q&A]
I have a technical question, is the video in the Gopro internally created from raw files, then processed in raw and saved to HEVC?

The other question is if I record 4K video instead of 5.3K, I know I lose quality, what is responsible for this? , what is this downscaling process called? David I will be very grateful for your answer, you are doing a great job👍.

**Comment by dnewman-gpsw** [comment]
I'm about to be away, so last question for while. 

The GP2 has a range of hardware accelerated scalers.  Sensor Res -> Scale 4K -> HEVC encode is setup to have more battery life, than 5.3K encoding. 5.3K encoding on camera, and scaling in post, is likely higher quality than scaling on camera (if you use a high BITR.)  All camera operations have a power-quality balance.  I'm guessing this what is driving your question.

**Reply by Pax709** [reply]
Yes, this I know that scaling in post-production will give better quality and that is what I do. It's a matter of fact-finding, whether Sensor Res -> it's “raw” ?   - Scale 4K - is it pixelskipping or lineskipping, or is it downscale?, someone wrote me that it's not downscale but just pixelskipping or lineskipping, that's why I want to clarify it and I don't have such knowledge.

**Reply by dnewman-gpsw** [reply]
Line skipping or pixel binning may happen in modes like 2.7k240, but not in 4k. You would see it as the res drop is not large. There is scaler just fewer taps than you would use in post.

**Reply by Pax709** [reply]
That is, when recording in 5.3, the camera does not perform an additional action -Scale ?
By the way, it would be nice to have 4K with higher quality, Gopro labs could conjure up something ? ;)

**Reply by dnewman-gpsw** [reply]
No scales, other than the higher quality resampling used by Hypersmooth and ERS compensation.
